ps_customeraccountlinks.php

```diff
@@ -100,33 +100,33 @@ public function getWidgetVariables($hookName = null, array $configuration = [])
 
         $my_account_urls = array(
             0 => array(
-                'title' => $this->l('Orders'),
+                'title' => $this->trans('Orders', array(), 'Admin.Global'),
                 'url' => $link->getPageLink('history', true),
             ),
             2 => array(
-                'title' => $this->l('Credit slips'),
+                'title' => $this->trans('Credit slips', array(), 'Modules.Customeraccountlinks.Admin'),
                 'url' => $link->getPageLink('order-slip', true),
             ),
             3 => array(
-                'title' => $this->l('Addresses'),
+                'title' => $this->trans('Addresses', array(), 'Shop.Theme'),
                 'url' => $link->getPageLink('addresses', true),
             ),
             4 => array(
-                'title' => $this->l('Personal info'),
+                'title' => $this->trans('Personal info', array(), 'Modules.Customeraccountlinks.Admin'),
                 'url' => $link->getPageLink('identity', true),
             ),
         );
 
         if ((int)Configuration::get('PS_ORDER_RETURN')) {
             $my_account_urls[1] = array(
-                'title' => $this->l('Merchandise returns'),
+                'title' => $this->trans('Merchandise returns', array(), 'Modules.Customeraccountlinks.Admin'),
                 'url' => $link->getPageLink('order-follow', true),
             );
         }
 
         if (CartRule::isFeatureActive()) {
             $my_account_urls[5] = array(
-                'title' => $this->l('Vouchers'),
+                'title' => $this->trans('Vouchers', array(), 'Modules.Customeraccountlinks.Admin'),
```

Comment:
Shop.Theme.Customeraccount (or CustomerAccount depending if we removed the camel case or not)